kolibri/core/assets/src/core-actions.js

```diff
@@ -376,4 +549,7 @@ module.exports = {
   updateProgress,
   saveLogs,
   samePageCheckGenerator,
+  initMasteryLog,
+  initAttemptLog,
+  saveAssessmentLogs,
```

Comment:
(Don't see an example of this in use - still trying to grok why it doesn't take any parameters and when it's meant to be called...)